deconv/figures/figure2.py

```diff
@@ -0,0 +1,33 @@
+from .common import subplotLabel, getSetup
+from ..imports import load_tables, load_figures, infer_x
+
+def makeFigure():
+    """Check the fit using infered x values"""
+    ax, f = getSetup((6, 3), (1, 2))
+
+    A_antiD, _, glycan_list = load_tables()
+    adcc_3a, adcc_3b = load_figures()
+
+    mean_3a = (adcc_3a.groupby(level=0).sum()) / 4
+    mean_3b = (adcc_3b.groupby(level=0).sum()) / 4
+
+    glycans_3a = infer_x(A_antiD, mean_3a)
+    glycans_3b = infer_x(A_antiD, mean_3b)
+
+    infer_adcc_3a = glycans_3a.dot(A_antiD)
```

Comment:
I'm getting an error that the shapes aren't aligned. You might want this. `@` is just another way to write matrix multiplication.

```suggestion
    infer_adcc_3a = A_antiD @ glycans_3a
```